src/databricks/labs/remorph/reconcile/query_builder_refactored/expression_generator.py

```diff
@@ -0,0 +1,159 @@
+from collections.abc import Callable
+from functools import partial
+
+from sqlglot import expressions as exp
+from sqlglot.expressions import (
```

Comment:
Please have consistency here, some expressions are, maintain consistency  
``` exp.Column ``` while others are ``` Trim ```

Reply:
updated.

Reply:
I still see return types are not updated.